instant/contract-deployments.md

```diff
@@ -2,21 +2,21 @@
 
 | Contract | Version                                                            | Address                                                                                                                 | Commit                                                                                                    |
 | :------- | :----------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------- |
-| Swap     | [`4.3.5`](https://www.npmjs.com/package/@airswap/swap/v/4.3.5)     | [`0x3E0c31C3D4067Ed5d7d294F08B79B6003B7bf9c8`](https://etherscan.io/address/0x3E0c31C3D4067Ed5d7d294F08B79B6003B7bf9c8) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Indexer  | [`3.6.7`](https://www.npmjs.com/package/@airswap/indexer/v/3.6.7)  | [`0xbA9aB9710Bd461F30C247f4cA2Cb7f453C22570e`](https://etherscan.io/address/0xbA9aB9710Bd461F30C247f4cA2Cb7f453C22570e) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Delegate | [`1.5.6`](https://www.npmjs.com/package/@airswap/delegate/v/1.5.6) | [`0xCA109988465266970bE8fd0048A7e235b9e62Bc3`](https://etherscan.io/address/0xCA109988465266970bE8fd0048A7e235b9e62Bc3) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Wrapper  | [`2.5.7`](https://www.npmjs.com/package/@airswap/wrapper/v/2.5.7)  | [`0xC4c01Afb1211E007CFA48f56552FB8a8B0Bdc51c`](https://etherscan.io/address/0xC4c01Afb1211E007CFA48f56552FB8a8B0Bdc51c) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Types    | [`2.4.4`](https://www.npmjs.com/package/@airswap/types/v/2.4.4)    | [`0xef5ED6D6f1600C56c54c51841bCF97421CDdb52d`](https://etherscan.io/address/0xef5ED6D6f1600C56c54c51841bCF97421CDdb52d) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
+| Swap     | [`5.4.7`](https://www.npmjs.com/package/@airswap/swap/v/5.4.7)     | [`0x4572f2554421Bd64Bef1c22c8a81840E8D496BeA`](https://etherscan.io/address/0x4572f2554421Bd64Bef1c22c8a81840E8D496BeA) | [`953956f`](https://github.com/airswap/airswap-protocols/commit/953956f308c65ec53d1f1b20d35f47fe04b936af) |
+| Indexer  | [`3.6.8`](https://www.npmjs.com/package/@airswap/indexer/v/3.6.8)  | [`0xbA9aB9710Bd461F30C247f4cA2Cb7f453C22570e`](https://etherscan.io/address/0xbA9aB9710Bd461F30C247f4cA2Cb7f453C22570e) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
```

Comment:
At the provided commit, the indexer version is 2.6.7.beta.2 not 3.6.8

Reply:
I think the commit and version we provide should match

Reply:
updated it to be https://github.com/airswap/airswap-protocols/commit/48da430ca23876b9d01743d59b548f7fb59660ab